corehq/form_processor/serializers.py

```diff
@@ -1,4 +1,4 @@
-from json_field import JSONField
+from jsonfield import JSONField
```

Comment:
It's because we're in the process of upgrading and the new version uses a
different package name.